### PR TITLE
Svg social icons & inverted icons Social + Audio

### PR DESCRIPTION
**Pull request by studiorabota**

### CODE REVIEW

File: app/views/pageflow/entries/_navigation.html.erb [hunk 1]
```diff
@@ -7,11 +7,39 @@
...
+    this.setInvert(page.element[0]);
+  },
+
+  setInvert: function(page_element) {
+    $(page_element).hasClass('invert') ? this.container.addClass('page-invert') : this.container.removeClass('page-invert');
+  },
+
+  init: function() {
+
+    // Init active dot
```
Comment (tilsammans) [inline app/views/pageflow/entries/_navigation.html.erb]: Old comment ;)

File: app/views/pageflow/entries/_navigation.html.erb [hunk 2]
```diff
@@ -7,11 +7,39 @@
 </div>
 
 <script>
+pageflow.Invert = {
```
Comment (tilsammans) [inline app/views/pageflow/entries/_navigation.html.erb:11]: I would like a one-line description of the class, why this class was created (what does it do).

**Comment by tilsammans** [comment]
Twee opmerkingen, kun je daar naar kijken? Dan zal ik mergen!

**Comment by studiorabota** [comment]
@tilsammans Comments verbeterd

**Pull request merged**